pkg/capacity/options.go

```diff
@@ -0,0 +1,20 @@
+package capacity
+
+// Option is a struct containing the command line options
+// FetchAndPrint depends on
+type Option struct {
```

Comment:
Tiny Nit: I think `Options` would be a bit more natural than `Option` here.
```suggestion
type Options struct {
```